rcnn/BatchedNms.cu

```diff
@@ -63,7 +63,7 @@ int batchedNms(int batch_size,
         workspace_size += get_size_aligned<float>(count);  // scores_sorted
 
         size_t temp_size_sort = 0;
-        thrust::cuda_cub::cub::DeviceRadixSort::SortPairsDescending(
```

Comment:
Why do we need to remove this namespace `thrust::cuda_cub::`?

Reply:
This should be an API interface change, where
#include <thrust/system/cuda/detail/cub/device/device_radix_sort.cuh>
change to
#include <cub/device/device_radix_sort.cuh>
Therefore, its namespace has changed, and the original path can no longer find the file